public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -111,68 +112,70 @@ export default class FindingDetailsFlyout extends Component<
         <div key={key}>
           <EuiAccordion
             id={`${key}`}
+            className="euiAccordionForm"
+            buttonClassName="euiAccordionForm__button"
             buttonContent={
               <div data-test-subj={'finding-details-flyout-rule-accordion-button'}>
-                <EuiText size={'s'}>{fullRule.title}</EuiText>
-                <EuiText size={'s'} color={'subdued'}>
-                  Severity: {severity}
-                </EuiText>
+                <EuiTitle size={'s'}>
+                  <h3>Rule details</h3>
+                </EuiTitle>
               </div>
             }
             initialIsOpen={rules.length === 1}
             data-test-subj={`finding-details-flyout-rule-accordion-${key}`}
           >
-            <EuiSpacer size={'m'} />
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                {/*//TODO: Refactor EuiLink to filter rules table to the specific rule.*/}
-                <EuiFormRow label={'Rule name'}>
-                  <EuiLink
-                    onClick={() => this.showRuleDetails(fullRule, rule.id)}
-                    data-test-subj={`finding-details-flyout-${fullRule.title}-details`}
-                  >
-                    {fullRule.title || DEFAULT_EMPTY_DATA}
-                  </EuiLink>
-                </EuiFormRow>
-              </EuiFlexItem>
+            <EuiPanel color="subdued">
+              <EuiFlexGroup>
+                <EuiFlexItem>
+                  {/*//TODO: Refactor EuiLink to filter rules table to the specific rule.*/}
```

Comment:
we can remove this TODO, since we show the Rules flyout for this link

Reply:
Removed.